schedule/sles4sap/sap_deployment_automation_framework/ensa2_scenarios.yml

```diff
@@ -16,6 +16,7 @@ schedule:
   - sles4sap/redirection_tests/ensa2_schedule_sapinstance_tests
   - sles4sap/redirection_tests/ensa2_hanadb_sapcontrol_move_ascs
   - sles4sap/redirection_tests/ensa2_hanadb_refresh_sapinstances
+  - sles4sap/redirection_tests/upload_logs
```

Comment:
Scheduling it like this won't work as intended. 
``` - sles4sap/redirection_tests/ensa2_schedule_sapinstance_tests ```
Schedules the sapinstance tests at the end of whole queue which means `upload_logs` won't run at the end of the whole sequence. 

Check you VR: 
https://openqaworker15.qa.suse.cz/tests/330148#step/upload_logs/1

Uploading logs happen before all tests. 
I am thinking that log collection could be part of cleanup job : 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/sles4sap/sap_deployment_automation_framework/sdaf_cleanup.yml

And it should probably go as well into post fail hook... but that might be a little more complicated.

Reply:
Sure, I will open a new ticket for this.